tests/integration/test_scmpipelinerun_view.py

```diff
@@ -325,10 +325,13 @@ def test_not_linked_at_all(
         assert str(another_another_scm_pipeline_run.public_identifier) in caplog.messages[0]
         assert str(another_another_scm_pipeline_run.first_parent_hash) in caplog.messages[0]
 
-    def test_linked_to_final_state(self, client, logged_in_user, application, scm_pipeline_run, next_scm_pipeline_run):
+    @pytest.mark.parametrize("status", [PIPELINE_STATUS_FAILED, PIPELINE_STATUS_SUCCESS, PIPELINE_STATUS_SKIPPED])
```

Comment:
I love that you made it parametrized to test all final statuses. Also good that you didn't take PIPELINE_FINAL_STATUSES (if someone removes SKIPPED, it will not test it anymore) although if we add another final status we might miss it here in this test.